config/patches/pkg-config-lite/pkg-config-lite-0.28-1.config.guess.patch

```diff
@@ -0,0 +1,992 @@
+--- config.guess_OLD	2015-10-30 21:16:33.910969812 +0000
++++ config.guess	2015-10-30 21:16:39.771905250 +0000
+@@ -1,14 +1,12 @@
+ #! /bin/sh
+ # Attempt to guess a canonical system name.
+-#   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
+-#   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010
+-#   Free Software Foundation, Inc.
++#   Copyright 1992-2015 Free Software Foundation, Inc.
+ 
+-timestamp='2009-12-30'
++timestamp='2015-10-21'
+ 
+ # This file is free software; you can redistribute it and/or modify it
+ # under the terms of the GNU General Public License as published by
+-# the Free Software Foundation; either version 2 of the License, or
++# the Free Software Foundation; either version 3 of the License, or
```

Comment:
Yeah but even if its all FUD, it still puts us in a situation where we have to 'chefsplain' to customers that what they believe is FUD, its vastly easier if we can just rip it all out, or as much out as possible, and then we don't even have to have those conversations with our customers...